db/migrate/20160105161736_create_reviews.rb

```diff
@@ -0,0 +1,6 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
```

Comment:
Unused block argument - `t`. You can omit the argument if you don't care about it.